recipes/libbezier/build.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# NOTE: This assumes the following environment variables have been set.
+#       - `${PREFIX}`
+#       - `${SRC_DIR}`
+# H/T: https://docs.conda.io/projects/conda-build/en/latest/user-guide/environment-variables.html
+
+mkdir -p build
+pushd build
+
+cmake \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DCMAKE_INSTALL_PREFIX:PATH="${PREFIX}" \
+    -DCMAKE_VERBOSE_MAKEFILE:BOOL=ON \
+    -DTARGET_NATIVE_ARCH:BOOL=OFF \
```

Comment:
I configured my CMake file to turn of `-march=native` when `TARGET_NATIVE_ARCH` is turned off.

I'm unclear if the goal is to produce shared libraries that can be moved from Linux-to-Linux (like a `manylinux` Python wheel) or if the goal is to produce a release build specific to the target architecture.

I'd **prefer** to leave `-march=native` turned on for the benefit of users, but am unsure if this is correct behavior for a `conda` package.

Reply:
I think you can just remove this line

Reply:
Yes, we have different builds for different architetures

Reply:
Sounds good, removed `-DTARGET_NATIVE_ARCH:BOOL=OFF` on all platforms.